src/lighthouse.php

```diff
@@ -253,6 +253,13 @@
          */
         'storage' => env('LIGHTHOUSE_SUBSCRIPTION_STORAGE', 'redis'),
 
+        /*
+         * Default subscription storage TTL.
+         *
+         * Indicates how long a subscription can be active before it's automatically removed from storage.
```

Comment:
```suggestion
         * Indicates how long a subscription can be active before it's automatically removed from storage.
         * Setting this to `null` means the subscriptions are stored forever. This may cause
         * stale subscriptions to linger indefinitely in case cleanup fails for any reason.
```